development/modules/students/servicesStudents.js

```diff
@@ -66,6 +120,13 @@
         })
     }
 
+    CourseStudentRelation.prototype.getAll = function () {
```

Comment:
Please, follow the standard in others relation builders. This method should be get and give you the option to use a custom filter. https://github.com/garusis/marint_web/blob/4496c9c47f7912f99ea9b12dcb20ca77258ea081/development/modules/students/servicesStudents.js#L86